redis/tests/support/sentinel.rs

```diff
@@ -118,10 +119,10 @@ impl RedisSentinelCluster {
         let mut servers = vec![];
         let mut folders = vec![];
         let mut master_ports = vec![];
-        let start_port = 7000;
 
-        for node in 0..masters {
-            let port = start_port + (node * (replicas_per_master + 1));
+        for _ in 0..masters {
+            let port = get_random_available_port();
+            sleep(Duration::from_millis(25));
```

Comment:
why do you sleep after getting the port number? and why don't you sleep after lines 135?

Reply:
Just forgot it there really, I was having an issue with the random ports so I put it there to see if it helped, but it turns out it was something else entirely. I'll do some clean up in those tests, I've also noticed I've left some commented lines in the "spawn" functions which can be removed.